rosdep/base.yaml

```diff
@@ -4774,6 +4781,7 @@ libssl-dev:
   gentoo: [dev-libs/openssl]
   nixos: [openssl]
   openembedded: [openssl@openembedded-core]
+  opensuse: [libopenssl-devel]
```

Comment:
I think this is the link for `libopenssl-devel` https://build.opensuse.org/package/show/openSUSE%3ALeap%3A15.2/openssl
The .spec file says
```
%package -n libopenssl-devel
```

The original `openssl-1_1` link seems to want the `-1_1` suffix, as the .spec file says
```
%package -n libopenssl-1_1-devel
```

Reply:
Ahh, good find @mabelzhang

Reply:
You can build a libopenssl-devel but they dont by default. libopenssl-devel is a meta package that pulls in the correct version of libopenssl devel. 

For openSUSE 15.2, that would be "libopenssl-1_1-devel". I figured that "libopenssl-devel" would be more future proof than "libopenssl-1_1-devel"

```neotinker@ros-15_2:~> zypper info --requires libopenssl-devel
Repository 'Main Update Repository' is out-of-date. You can run 'zypper refresh' as root to update it.
Repository 'Update Repository (Non-Oss)' is out-of-date. You can run 'zypper refresh' as root to update it.
Loading repository data...
Reading installed packages...


Information for package libopenssl-devel:
-----------------------------------------
Repository     : Main Repository
Name           : libopenssl-devel
Version        : 1.1.1d-lp152.1.1
Arch           : noarch
Vendor         : openSUSE
Installed Size : 246 B
Installed      : No
Status         : not installed
Source package : openssl-1.1.1d-lp152.1.1.src
Summary        : Include Files and Libraries mandatory for Development
Description    : 
    This package contains all necessary include files and libraries needed
    to develop applications that require these.
Requires       : [4]
    pkgconfig
    libopenssl1_1 = 1.1.1d
    libopenssl-1_1-devel = 1.1.1d
    openssl = 1.1.1d

neotinker@ros-15_2:~>

Reply:
Thanks for the explanation!